### PR TITLE
Add Visual Studio integration through `docfx.console` documentation

### PR DESCRIPTION
**Pull request by laedit**
Add info of `docfx.console` to the getting started page, along with two possible warnings and the way to fix them.

### CODE REVIEW

File: Documentation/tutorial/docfx_getting_started.md [hunk 1]
```diff
@@ -41,7 +41,32 @@ docfx docfx_project\docfx.json --serve
...
 Now you can view the generated website on http://localhost:8080.
-3. Use *DocFX* with a Build Server
+
+3. Use *DocFX* integrated with Visual Studio
+---------------
+
+*Step1.* Install the [`docfx.console`](https://www.nuget.org/packages/docfx.console/) (formerly `docfx.msbuild`) nuget package on the project that you want to document.
+It add itself to the build targets and add the `docfx.json` configuration file along with other files.
+
+*Step2.* Compile, a `_testsite` folder is generated with the documentation.
```
Comment (vicancy) [inline Documentation/tutorial/docfx_getting_started.md]: `_site` folder?

File: Documentation/tutorial/docfx_getting_started.md [hunk 2]
```diff
@@ -41,7 +41,32 @@ docfx docfx_project\docfx.json --serve
...
+
+> [!NOTE]
+> *Possible warnings*:
+> - *No metadata is generated for _project_name_*: *DocFx* is still dependent to Microsoft Build Tools 2015, so If you haven't VS 2015 installed, you need at least the [Microsoft Build Tools 2015](https://www.microsoft.com/en-us/download/details.aspx?id=48159)
```
Comment (vicancy) [inline Documentation/tutorial/docfx_getting_started.md]: The Build Tools 2015 dependency is removed now.

**Comment by laedit** [comment]
Thanks for the review, I haven't seen that a new version have been out 😃

File: Documentation/tutorial/docfx_getting_started.md [hunk 3]
```diff
@@ -41,7 +41,31 @@ docfx docfx_project\docfx.json --serve
...
+---------------
+
+*Step1.* Install the [`docfx.console`](https://www.nuget.org/packages/docfx.console/) (formerly `docfx.msbuild`) nuget package on the project that you want to document.
+It add itself to the build targets and add the `docfx.json` configuration file along with other files.
+
+*Step2.* Compile, a `_site` folder is generated with the documentation.
+
+> [!NOTE]
+> *Possible warning*:
+> - *Cache is corrupted*: if your project targets multiple frameworks, you have to indicate one to be the main for the documentation, through the [`TargetFramework` property](https://github.com/dotnet/docfx/issues/1254#issuecomment-294080535) in `docfx.json`:
```
Comment (vicancy) [inline Documentation/tutorial/docfx_getting_started.md]: *Cache is corrupted* should be a false warning, a bug to fix 😄 I will update the document when the bug is fixed..

**Pull request merged**